.gitignore

```diff
@@ -47,3 +47,4 @@ testem.log
 # System Files
 .DS_Store
 Thumbs.db
+/projects/ngx-skeleton-loader/node_modules/
```

Comment:
Could you elaborate why's that required, please?

Reply:
Removed. Think the IDE was trying to be smart here.